custom_components/smartthinq_sensors/wideq/core_async.py

```diff
@@ -146,7 +146,7 @@ def _oauth_info_from_result(result_info: dict) -> dict:
 
 def lg_client_session() -> aiohttp.ClientSession:
     """Create an aiohttp client session to use with LG ThinQ."""
-    context = ssl.create_default_context()
+    context = get_default_context()
     context.set_ciphers(_LG_SSL_CIPHERS)
```

Comment:
This will mutate the default ssl context

Reply:
The simplest fix would be to create the context and store it in a constant so it doesn't happen at run time, and instead at import time which will run in the executor

Reply:
Thanks!!!

Reply:
Adjusted as suggested, yes this works without any warnings.

Reply:
Thanks @iprak 👍

I'm far from home for work this month, I will work to merge this PR and prepare next release probably by the end of next week.